reference/forms/types/password.rst

```diff
@@ -76,4 +76,10 @@ The default value is ``''`` (the empty string).
 
 .. include:: /reference/forms/types/options/required.rst.inc
 
-.. include:: /reference/forms/types/options/trim.rst.inc
+trim
+~~~~
+**type**: ``Boolean`` **default**: ``false``
```

Comment:
I added the blank lines but I prefer to let Boolean rather than bool for the moment as all the others references in the documentation use Boolean.